app/build.gradle

```diff
@@ -42,9 +47,13 @@ dependencies {
     implementation "com.squareup.moshi:moshi-adapters:1.11.0"
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
     implementation "androidx.recyclerview:recyclerview:1.1.0"
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```

Comment:
would be good to see if this can be removed, otherwise, interested in what this is being used for!

Reply:
had a quick look at the androidx documentation, looks like this has been depreciated so can be deleted (on a side note, not sure why AS added this)